.nvmrc

```
@@ -0,0 +1 @@
+v18.15.0
```

```suggestion
18
```
Just `18` should be fine here - I don't believe there are any issues running on other v18.x versions